src/components/input-switch/input-field.tsx

```diff
@@ -57,6 +57,23 @@ export type InputFieldProps = {
    * by default, we are capturing the "enter" key event and stopping it
    */
   allowEnter?: boolean
+  /**
+   * true if the current Input field is an element of an InputField of type array
+   */
+  isArrayElement? : boolean
```

Comment:
Thanks for your adding this prop. But let's call it based on the effect that it does, not where you're using this. If you can come up with a better name than `displayRequireErrorBeforeSubmit` please feel free to change it. Otherwise use this one.

```ts
/**
 * display required error regardless of whether the form has been submitted or not.
 */ 
displayRequireErrorBeforeSubmit?: boolean
```